static/js/script.js

```diff
@@ -0,0 +1,62 @@
+const form = document.getElementById('frm_books');
+const display = document.getElementById('display');
+
+function saveBooks(data) {
+  const strData = JSON.stringify(data);
+  localStorage.setItem('data', strData);
+}
+
+let catalog = {};
+
+function restoreBooks() {
+  const data = localStorage.getItem('data');
+  const objData = JSON.parse(data);
+  catalog = objData || {};
+}
+
+function displayData(k) {
+  const bookParent = document.createElement('div');
+  bookParent.classList.add(`parent-${k}`);
+  bookParent.classList.add('book-parent');
+  bookParent.innerHTML += catalog[k].title;
+  bookParent.innerHTML = `${bookParent.innerHTML}<br>${catalog[k].author}`;
+  bookParent.innerHTML = `${bookParent.innerHTML}<br> <button id="${k}"> Remove </button>`;
+  display.appendChild(bookParent);
+  saveBooks(catalog);
+}
+
+function removeBook(a) {
+  const remove = document.getElementById(a);
+  if (remove) {
+    remove.addEventListener('click', function removeContent(event) {
+      const button = event.target;
+      const parent = button.parentNode;
+      parent.parentNode.removeChild(parent);
+      delete catalog[this.id];
+      saveBooks(catalog);
+    });
+  }
+}
```

Comment:
[OPTIONAL]
- [ ] I Suggest that you make use of a unique id property for each book since it would make it easier to remove books with duplicate values.
- [ ] you can also favor the use array methods that doesn't mutate the array, e.g. `concat`, `filter`, `slice` instead of `push`, `pop`, `splice` and so on
- > code examples
```javascript
const books = [];

const removeBook = (bookID) => {
  // bookID can be any other property, e.g. title or author, in as much it is unique
  books = books.filter(({ id }) => bookID !== id); // This assumes that `bookID` parameter is the book id
};
```